aptos-move/block-executor/src/scheduler.rs

```diff
@@ -138,6 +138,7 @@ enum ExecutionStatus {
     Executing(Incarnation),
     Suspended(Incarnation, DependencyCondvar),
     Executed(Incarnation),
+    // TODO: rename to Finalized or ReadyToCommit / CommitReady? it gets committed later, without scheduler tracking.
```

Comment:
RWFinalized, or maybe RW is a detail, but it also represents that the handling depends on what we want to do w. non-RW things

Reply:
leaving for a separate PR